static/bidder-info/missena.yaml

```diff
@@ -12,5 +12,5 @@ capabilities:
       - banner
 userSync:
   iframe:
-    url: https://sync.missena.io/iframe?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
-    userMacro: $UID
\ No newline at end of file
+    url: https://sync.missena.io/iframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&t=YOUR_API_KEY&redirect={{.RedirectURL}}
```

Comment:
Is an api key from Missena now required for user syncing?

Reply:
yes

Reply:
Thank you for confirming. It's not appropriate to define user syncing which does not work out of the box. Please update this config to indicate that user syncing is possible, but requires configuring with a Missena provided api key. To signal availability of user syncing to host companies, [please use the supports attribute documented here](https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#user-sync-configuration).

You're welcomed to keep the url and usermacro definitions if they're commented out.